### PR TITLE
Login setup

### PR DESCRIPTION
**Pull request by Iqbal-Elham**
## In this pull I implemented the following requirements:

- The user logs in to the website, only by typing the username.
- Design and implement login page
capture1
Capture2

### CODE REVIEW

**Comment by Muhammad0602** [comment]
@Iqbal-Elham why didn't you add the password and email.

**Comment by Muhammad0602** [comment]
What will happen if we have 2 users with the same username, will we ask them to change their name? 😁

**Review by Muhammad0602 — CHANGES_REQUESTED** [review]
Hi @Iqbal-Elham  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- The app works as expected ✔️ 
- Good UI ✔️ 
- Linter checks pass


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/loginpage/Login.js [hunk 1]
```diff
@@ -1,9 +1,36 @@
...
+            src="https://cdn.lordicon.com/ljvjsnvh.json"
+            trigger="loop"
+            delay="1000"
+            colors="primary:#16c72e,secondary:#242424"
+            state="morph"
+            style={{ width: '180px', height: '180px' }}
+          />
+          <input type="text" value={username} onChange={(e) => setUsername(e.target.value)} placeholder="Username" />
+          <button type="submit">Log In</button>
+        </form>
```
Comment (Muhammad0602) [inline src/components/loginpage/Login.js:30]: You should consider adding a field for the email and password for the login and password confirmation for sign up.

**Comment by Muhammad0602** [comment]
Also I noticed that when the user is signing in or login he has access to entire navigation panel, which should not be the case. I think @mrkamin should fix that as it is his part.
@Muhammad0602 @Iqbal-Elham 
the issue fixed now the access made limt.

**Comment by Iqbal-Elham** [comment]
Hi @Muhammad0602, hope you are doing great, Thank you for reviewing my pull request and your nice feedback. In case of email and password I think it is not required for us because we are group of 4 and in case of username duplicating I fix the issue you can checkout it out again whenever you had time. ❤️ 🙏🏻

**Review by mrkamin — APPROVED** [review]
Hello @Iqbal-Elham  ,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Review by Muhammad0602 — APPROVED** [review]
Hi @Iqbal-Elham, 
Great job, your project is now complete.

**Pull request merged**